lib/src/html_generator.dart

```diff
@@ -41,10 +41,10 @@ class Templates {
   TemplateRenderer topLevelPropertyTemplate;
   TemplateRenderer typeDefTemplate;
 
-  final Map<String, String> _partialTemplates = {};
+  final Map<String, String> _partialTemplates = <String, String>{};
```

Comment:
why do we need the generics here? how does it help?  (do we ever do an `is` check on this object?)